### ISSUE TITLE
throw new Error('Non-base' + BASE + ' character')

### ISSUE THREAD

**Issue by shortthefomo** [Mwni/xrpl-bookworm#1]
![Screen Shot 2023-07-15 at 23 10 28](https://github.com/Mwni/xrpl-bookworm/assets/508629/103780b0-8e03-441c-8392-3f124198373f)

**Comment by Mwni** [comment]
I think that the `@xrplkit` dependencies have changed since then are no longer compatible. Let me push an update with pinned versions.

**Comment by shortthefomo** [comment]
yeah know the base libs in flux ever where atm, no rush off to bed here.

**Comment by Mwni** [comment]
Turns out the dependencies are still fine, but you did not supply an operator wallet seed in `config.json`. I added a check for that https://github.com/Mwni/xrpl-bookworm/commit/b368e4b9a7141cbfe6e22d2047c252cbf88ba925
